### ISSUE TITLE
Remove "install button color" A/B experiment

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-frontend#7486]
We introduced an A/B experiment in #6144. The aim of this experiment was to determine the best color for the install button (blue vs green). According to the Analytics results, there is no significant difference. I suggest we remove this experiment and use the initial color for the install button: blue.

**For QA:** the install button should be blue, never green.

**Comment by twisted-sres** [comment]
@willdurand can I take this up?

**Comment by willdurand** [comment]
> @willdurand can I take this up?

@twisted-sres sure but I won't have much time to help you with this one unfortunately. You can find other issues by looking at the [opened `contrib: welcome` issues](https://github.com/mozilla/addons-frontend/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22contrib%3A+welcome%22).

**Comment by willdurand** [comment]
@atsay @jvillalobos before moving forward, are you OK with that? I'd like to remove this experiment this week if possible (there is already an approved patch from a contributor for that).

Stats from November 1st below:

![screen shot 2019-01-28 at 12 34 07](https://user-images.githubusercontent.com/217628/51833923-1077d280-22f9-11e9-8746-1bfd08e87f1b.png)

**Comment by ioanarusiczki** [comment]
@willdurand I verified it on AMO dev - FF64 (Win10).  

- Even for older profiles if the cookie was set to green, the button is displayed as blue (as in the screenshot below), then I cleared all cookies and the `installButtonColorExperiment` cookie was no longer available. 

![blue is back](https://user-images.githubusercontent.com/33448286/51910920-2bba0f00-23d9-11e9-9e4c-b241c42df4f8.png)

- On all new profiles the cookie is no longer there and the buttons are blue.  

This is AMO dev versus AMO stage

![no cookie for the bluegreenbutton](https://user-images.githubusercontent.com/33448286/51910935-37a5d100-23d9-11e9-9cee-a2976683cf87.gif)
